activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -2896,51 +2922,142 @@ public void setSslContext(SslContext sslContext) {
      */
+    @Deprecated
     public void setWaitForSlave(boolean waitForSlave) {
         this.waitForSlave = waitForSlave;
+        this.waitForStandby = waitForSlave;
+        LOG.warn("Configuration `waitForSlave` is deprecated and will be removed in a future release. Use `waitForStandby` instead.");
     }
 
+    /**
+     * @deprecated AMQ-7514 in the move to inclusive nomenclature
+     */
+    @Deprecated
     public long getWaitForSlaveTimeout() {
         return this.waitForSlaveTimeout;
     }
 
+    /**
+     * Configuration `waitForSlaveTimeout` is deprecated and will be removed in a future release. Use `waitForStandbyTimeout` instead.
+     * @deprecated AMQ-7514 in the move to inclusive nomenclature
+     * @param waitForSlaveTimeout
+     */
+    @Deprecated
     public void setWaitForSlaveTimeout(long waitForSlaveTimeout) {
         this.waitForSlaveTimeout = waitForSlaveTimeout;
+        this.waitForStandbyTimeout = waitForSlaveTimeout;
+        LOG.warn("Configuration `waitForSlaveTimeout` is deprecated and will be removed in a future release. Use `waitForStandbyTimeout` instead.");
+
+    }
+
+    public boolean isWaitForStandby() {
+        return waitForStandby;
     }
 
     /**
-     * Get the passiveSlave
+     * @org.apache.xbean.Property propertyEditor="org.apache.activemq.util.BooleanEditor"
+     */
+    public void setWaitForStandby(boolean waitForStandby) {
+        this.waitForStandby = waitForStandby;
+        this.waitForSlave = waitForStandby;
+    }
+
+    public long getWaitForStandbyTimeout() {
+        return this.waitForStandbyTimeout;
+    }
+
+    public void setWaitForStandbyTimeout(long waitForStandbyTimeout) {
+        this.waitForSlaveTimeout = waitForStandbyTimeout;
+        this.waitForStandbyTimeout = waitForStandbyTimeout;
+    }
+
+    /**
+     * Configuration `passiveSlave` is deprecated and will be removed in a future release. Use `standbyInstance` instead.
+     * @deprecated AMQ-7514 in the move to inclusive nomenclature
      * @return the passiveSlave
      */
+    @Deprecated
     public boolean isPassiveSlave() {
+        LOG.warn("Configuration `passiveSlave` is deprecated and will be removed in a future release. Use `standbyInstance` instead.");
         return this.passiveSlave;
     }
 
     /**
-     * Set the passiveSlave
+     * Configuration `passiveSlave` is deprecated and will be removed in a future release. Use `standbyInstance` instead.
+     * @deprecated AMQ-7514 in the move to inclusive nomenclature
      * @param passiveSlave the passiveSlave to set
      * @org.apache.xbean.Property propertyEditor="org.apache.activemq.util.BooleanEditor"
      */
+    @Deprecated
     public void setPassiveSlave(boolean passiveSlave) {
         this.passiveSlave = passiveSlave;
+        this.standbyInstance = passiveSlave;
+        LOG.warn("Configuration `passiveSlave` is deprecated and will be removed in a future release. Use `standbyInstance` instead.");
```

Comment:
@mattrpav per your comment it also appears `isPassiveSlave` is not used. We could also remove this it seems like?

Reply:
Yes, I think we break-up the PR's a bit. Removing deprecated methods can be its own thing